simulation/tools/sim_viewer.cpp

```diff
@@ -446,8 +446,8 @@ void capture (Eigen::Isometry3d pose_in, string point_cloud_fname)
   pcl::PointCloud<pcl::PointXYZRGB>::Ptr pc_out (new pcl::PointCloud<pcl::PointXYZRGB>);
   bool write_cloud = true;
   
-  if (write_cloud)
-  {
+  /*if (write_cloud)
+  {*
```

Comment:
In this file as well.

Reply:
Ok, I removed a lot of more disabled code of this file now, too.